src/mbgl/util/logging.cpp

```diff
@@ -24,11 +25,20 @@ class Log::Impl {
         : scheduler(Scheduler::GetSequenced()) {}
 
     void record(EventSeverity severity, Event event, int64_t code, const std::string& msg) {
```

Comment:
You can use a function try block here. I.e.

```
void fun()
try {
}
catch(...) {
}
```

instead of

```
void fun() {
try {
}
catch(...) {
}
```